Std/Data/Bool.lean

```diff
@@ -110,6 +117,9 @@ theorem or_eq_true_iff : ∀ (x y : Bool), (x || y) = true ↔ x = true ∨ y =
 theorem or_eq_false_iff : ∀ (x y : Bool), (x || y) = false ↔ x = false ∧ y = false := by
   decide
 
+/-! non infix aliases -/
```

Comment:
Note that in mathlib we often use the infix-like naming scheme for regular functions that are "morally" infix even if there is no infix notation.

Reply:
Yes, I'm aware. See my other comment.